<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="configWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::Reset|QDialogButtonBox::StandardButton::RestoreDefaults</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="txTab">
      <attribute name="title">
       <string>TX</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0" columnstretch="0,1">
       <item row="0" column="1">
        <widget class="QComboBox" name="txDeviceValue"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="txBufferSizeLabel">
         <property name="text">
          <string>Buffer size</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="txChannelsValue">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="txDeviceLabel">
         <property name="text">
          <string>Device</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="txSamplerateLabel">
         <property name="text">
          <string>Sample rate</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="txChannelsLabel">
         <property name="text">
          <string>Channels</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="txSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="txSamplerateValue">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="txFormatValue">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="txFormatLabel">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="txBufferSizeValue"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="monitorTab">
      <attribute name="title">
       <string>Monitor</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,1,0" columnstretch="0,1">
       <item row="4" column="1">
        <widget class="QComboBox" name="monitorChannelsValue">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="monitorSamplerateValue">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="monitorDeviceValue"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="monitorDeviceLabel">
         <property name="text">
          <string>Device</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="monitorChannelLabel">
         <property name="text">
          <string>Channels</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="monitorFormatLabel">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="monitorFormatValue">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="monitorSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>32</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="monitorSamplerateLabel">
         <property name="text">
          <string>Sample rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="monitorBufferSizeLabel">
         <property name="text">
          <string>Buffer size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="monitorBufferSizeValue"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="morseinoTab">
      <attribute name="title">
       <string>Morseino</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
       <item row="0" column="0">
        <widget class="QLabel" name="morseinoSerialPortName">
         <property name="text">
          <string>Serial Port</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="morseinoSerialPortValue"/>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>146</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
